sql/bigistore.sql

```diff
@@ -68,6 +68,16 @@ CREATE FUNCTION bigistore(integer[])
     AS 'istore', 'bigistore_from_intarray'
     LANGUAGE C IMMUTABLE STRICT;
 
+CREATE FUNCTION bigistore_eq(bigistore, bigistore)
+    RETURNS bool
+    AS 'istore'
+    LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE FUNCTION bigistore_not_eq(bigistore, bigistore)
```

Comment:
a little side note the postgres convention is `_ne` for not equal functions and `_eq` for equality
would make grep'ing through the code base easier